src/items/functions/item/item_parse.cpp

```diff
@@ -563,6 +563,8 @@ void ItemParse::parseAbsorbPercent(const std::string &tmpStrValue, pugi::xml_att
 		itemType.getAbilities().absorbPercent[combatTypeToIndex(COMBAT_FIREDAMAGE)] += pugi::cast<int16_t>(valueAttribute.value());
 	} else if (stringValue == "absorbpercentpoison") {
```

Comment:
can be used? `} else if (stringValue == "absorbpercentpoison" || stringValue == "absorbpercentearth") {`